test/test_jit.py

```diff
@@ -3051,6 +3051,8 @@ def weighted_kernel_sum(self, weight):
             check_forward_input = torch.rand(1, 1, 3, 3)
             check_inputs.append({'forward' : check_forward_input, 'weighted_kernel_sum' : check_weight})
         module = torch.jit.trace_module(n, inputs, True, check_inputs)
+        self.assertTrue(module._c._has_method("forward"))
```

Comment:
actually run method maybe ?

Reply:
`trace_module` should be running and checking it.
https://github.com/pytorch/pytorch/blob/711be82951739e8c3cf7e5d2f81d22e93451fa11/torch/jit/__init__.py#L899